projects/plugins/jetpack/changelog/update-defensive-is-onboarding

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: compat
+Comment: I already mentioned the change in another changelog.
```

Comment:
I think I would add an entry here anyway, but this time targetting the Jetpack plugin users. Maybe something explaining how this will avoid Fatal errors when Jetpack is used alongside other plugins using shared features, something like that?

Reply:
Added, thank you!

Reply:
Added, thank you!